pkg/operation/shoot/types.go

```diff
@@ -43,6 +43,7 @@ type Shoot struct {
 	WantsAlertmanager      bool
 	IgnoreAlerts           bool
 	HibernationEnabled     bool
+	EnableContainerD	   bool
```

Comment:
Do we really need this field? It's not very useful, is it?

Reply:
I do use this field to render the OSC.
operatingsystemconfig.go line 92

Reply:
I know. But usually the `Shoot` structs only contains fields that are "expensive" to re-compute. In this case you only read a value from the Shoot manifest, so there is no reason/need to store it as struct attribute.

Reply:
done